### DISCUSSION TITLE
Make action colum not exportable by default

### DISCUSSION THREAD

**Discussion by MrYamous** [Power-Components/livewire-powergrid#1737] [Ideas]
Due to their nature (perform an action on data from Powergrid table), I think in most cases action column do not need to be present in exports.
Then i suggest to hide those column from export by default, what do you think about it ? 

I can work on PR for that but prefer to ask for your opinion before

**Comment by luanfreitasdev** [comment]
I think this is good

**Comment by luanfreitasdev** [comment]
Released: https://github.com/Power-Components/livewire-powergrid/releases/tag/v6.0.5
